MAINTAINERS.md

```diff
@@ -16,7 +16,7 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 | Peter Fitzgibbons | [pjfitzgibbons](https://github.com/pjfitzgibbons)   | Amazon      |
 | Simeon Widdis     | [swiddis](https://github.com/swiddis)               | Amazon      |
 | Chen Dai          | [dai-chen](https://github.com/dai-chen)             | Amazon      |
-| Vamsi Manohar     | [vamsi-amazon](https://github.com/vamsi-amazon)     | Amazon      |
```

Comment:
This was causing the `linkchecker` pipeline job to fail...

Reply:
https://github.com/vmcircuit also doesn't go anywhere

Reply:
Maybe just remove the link?

Reply:
Very weird. I swear I went to this address yesterday 🤔 
Anyway, just removed the link 🤦

Reply:
@currantw plz use this: https://github.com/vmmusings

Reply:
Thanks! Updated.